bee-message/src/output/signature_locked_single.rs

```diff
@@ -3,27 +3,27 @@
 
 use crate::{address::Address, error::ValidationError, MessageUnpackError, IOTA_SUPPLY};
 
-use bee_packable::{
-    error::{UnpackError, UnpackErrorExt},
-    packer::Packer,
-    unpacker::Unpacker,
-    Packable,
-};
+use bee_packable::{bounded::BoundedU64, Packable};
 
 use core::ops::RangeInclusive;
 
 /// Valid amounts for a signature locked single output.
 pub const SIGNATURE_LOCKED_SINGLE_OUTPUT_AMOUNT: RangeInclusive<u64> = 1..=IOTA_SUPPLY;
 
+pub(crate) type Amount =
+    BoundedU64<{ *SIGNATURE_LOCKED_SINGLE_OUTPUT_AMOUNT.start() }, { *SIGNATURE_LOCKED_SINGLE_OUTPUT_AMOUNT.end() }>;
```

Comment:
Is there some way we can simplify this a little? 😬

Reply:
we could introduce `SIGNATURE_LOCKED_SINGLE_OUTPUT_(MIN|MAX)_AMOUNT` but I'm not sure if that's simpler.